src/Futhark/CodeGen/Backends/GenericC.hs

```diff
@@ -1170,7 +1170,7 @@ onEntryPoint get_consts fname (Function _ outputs inputs _ results args) = do
          if (!($exp:(allTrue entry_point_input_checks))) {
            ret = 1;
            if (!ctx->error) {
-             ctx->error = msgprintf("Error: entry point arguments have invalid sizes.");
+             ctx->error = msgprintf("Error: entry point arguments have invalid sizes.\n");
```

Comment:
Why is this necessary?

Reply:
Emscripten has really weird behaviour when prints statements aren't ended with new line characters

This created a couple failed tests in the futhark test suite, for example in emptydim0.fut this test failed with unexpected behaviour. Specifically these sort of test cases below.

```futhark
-- ==
-- compiled input { [[1]] [[1,2]] } error: .
-- compiled input { [[1,2]] [[1]] } error: .

let main [n] (xs: [][n]i32) (ys: [][n]i32) = (xs, ys)
````

Reply:
It's not just a matter of `fflush()`ing the error stream?

Reply:
I'll add the `\n` on master, since it should be there anyway.